move:
  action_name: /admittance_controller/admittance
approach:
  action_name: /approach_controller/approach
  approach_frame: yumi_link_7_r
ft_reset_services: [/r_compensate_gravity/calibrate_bias, /l_compensate_gravity/calibrate_bias]
right_arm:
  pose: [0.473, -0.125, 0.148, 0.246, 0.663, 0.684, 0.181]
left_arm:
  pose: [0.403, 0.305, 0.173, 0.678, 0.263, 0.235, 0.644]
